### DISCUSSION TITLE
mixcr-postanalysis, v gene usage

### DISCUSSION THREAD

**Discussion by singhd6** [milaboratory/mixcr#1874] [Q&A] [answered]
i want to understand how do we calculate the v gene usage in the following command, what does weight means in the pdf plot generated, because i did not define the weight function, and the weight function was none
![image](https://github.com/user-attachments/assets/bb00978a-9ead-4d22-bc38-b3cc9b67e482)
![image](https://github.com/user-attachments/assets/a8be9813-39dd-4695-b3cd-294650f5791e)
1. what does value means in the output individual_result.vusage.IGH.tsv table
2. individual_result.vUsage.IGH.tsv 
![vUsage IGH (1)](https://github.com/user-attachments/assets/eee2da84-72d4-44fa-b8a9-76f0feebdf2f)
thank you 
divya

**Comment by mizraelson** [comment, marked as answer]
weight defines whether you want to weight the data by the number of reads. If weight is set to none, the numbers represent the fraction of clones with a certain V gene, regardless of their abundance in the repertoire.
